Source/Core/CoplanarPolygonGeometry.js

```diff
@@ -334,7 +359,17 @@ CoplanarPolygonGeometry.pack = function (value, array, startingIndex) {
   startingIndex += VertexFormat.packedLength;
 
   array[startingIndex++] = value._stRotation;
-  array[startingIndex] = value.packedLength;
+  if (value._textureCoordinates) {
+    startingIndex = PolygonGeometryLibrary.packPolygonHierarchy(
+      value._textureCoordinates,
+      array,
+      startingIndex,
+      Cartesian2
+    );
+  } else {
+    array[startingIndex++] = -1.0;
```

Comment:
I understand what this is doing, but I'm not sure if it would be the recommended approach. @lilleyse Do you have any thoughts here for setting a flag like this for packing/unpacking an optional value?